infra/templates/az-svc-data-integration-mlw/README.md

```diff
@@ -0,0 +1,139 @@
+# Azure Application Services
```

Comment:
Is this README correct? Looks the same as some of the app service environments but I don't see reference to any data factory integration in the document